core/jobs/playtime_entries.py

```diff
@@ -218,3 +231,52 @@ def _fetch_report_records(self, start: datetime, until: datetime) -> Query:
                 PlaytimeSummary.identifier_id,
             )
         )
+
+    @staticmethod
+    def _isbn_for_identifier(
+        identifier: Optional[Identifier],
+        /,
```

Comment:
Minor: The positional and name only arguments seem unneeded, but don't cause any hindrance either.

Reply:
I've been thinking about using these more to help ensure that the methods are called as expected. Would be interested in your thoughts -- either here or elsewhere -- on doing this.

Reply:
That's fine, was just curious for the use of the style :+1: